tox.ini

```diff
@@ -53,6 +53,7 @@ envlist =
     kafka-messagebroker_confluentkafka-py39-confluentkafka{0108},
     kafka-messagebroker_kafkapython-py38-kafkapython{020001,020000},
     kafka-messagebroker_kafkapython-{py37,py38,pypy310}-kafkapythonlatest,
```

Comment:
This isn't specific to this PR but I wonder if we should actually be testing the latest version of kafka-python on pypy310 since it only lists support up to Python3.8. It appears to be working but it doesn't seem correct to me.